### PR TITLE
expression, plan: support builtin aggregation function 'bit_xor'

### PR DESCRIPTION
**Pull request by spongedu**
for #5077, support builtin aggregation function 'bit_xor' in TiDB.
To support this pr, tipb must be updated too. See My another pr: https://github.com/pingcap/tipb/pull/46/files

### CODE REVIEW

File: _vendor/src/github.com/pingcap/tipb/go-tipb/expression.pb.go [hunk 1]
```diff
@@ -79,6 +79,7 @@ const (
 	ExprType_Max         ExprType = 3005
 	ExprType_First       ExprType = 3006
 	ExprType_GroupConcat ExprType = 3007
+	ExprType_AggBitXor ExprType = 3008
```
Comment (zz-jason) [inline _vendor/src/github.com/pingcap/tipb/go-tipb/expression.pb.go]: keep the code alignment.

Comment (spongedu) [reply _vendor/src/github.com/pingcap/tipb/go-tipb/expression.pb.go]: ok

**Comment by zz-jason** [comment]
LGTM

**Pull request merged**